cluster/node-pools/master-default/userdata.yaml

```diff
@@ -928,6 +928,7 @@ write_files:
       {{- end}}
 {{- end}}
 
+  # TODO: Remove this once all nodes are running an AMI compatible with /etc/cni/net.d/10-flannel.conflist
```

Comment:
Do we cover this in some internal issue? Would be better recorded there. I'm guessing we have an issue for the VPC CNI work?

Reply:
I created an internal issue: 3756

Reply:
Thanks 🙏